packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -518,10 +519,10 @@ const inputString: InputStringTest = async ({ context, path, value }): Promise<L
 };
 
 // Test input string widget
-export const inputStringTest: InputStringTest = ({ context, path, value }) => {
+export const inputStringTest: InputStringTest = ({ context, path, value, expectedValue }) => {
```

Comment:
Not required, but while you are at it, could you add a function header/description like the other function under it ?